tests/modules/services/window-managers/i3/i3-workspace-output.nix

```diff
@@ -0,0 +1,41 @@
+{ config, lib, ... }:
+
+with lib;
+
+{
+  config = let
+    i3 = {
+      ws1 = "1: ABC";
+      ws2 = "2: DEF";
+      ws3 = "3: XYZ";
```

Comment:
Could we have a workspace with just a number, and one with just text, to ensure it all works?

Reply:
Done, added cases for:
* Number only
* Text only
* Number: Text
* Special Chars

also updated sway-test according.